pages/gameplay/es6-basics/game-results.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+
+<html lang=" en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Harshna Haswani">
+  <meta name="description" content="Test yourself and grow into a rockstar developer">
+  <title> RockStar Dev — Game Results</title>
+
+  <link rel="stylesheet" href="/styles/main.css">
+
+</head>
+
+<body class="gameplay score ">
+
+
```

Comment:
remove unwanted spaces where you have given you can use comments to keeo a track, rest looks good to merge